mobile/src/pages/Login.tsx

```diff
@@ -0,0 +1,54 @@
+import { StatusBar } from 'expo-status-bar';
+import React from 'react';
+import { useState } from 'react';
+import { Button, TextInput, Text, Pressable, View } from 'react-native';
+import Icon from 'react-native-vector-icons/Ionicons';
+
+export default function Login() {
+  const [username, setUsername] = useState("");
+  const [password, setPassword] = useState("");
+
+  return (
+    <View className={'flex-1 w-full'}>
+      <View className={'w-full flex-row place-content-start mt-16 ml-2'}>
+        <Pressable>
+          <Icon size={45} name="arrow-back-outline" />
```

Comment:
Raciel likes to always wrap his strings in {}. I prefer not to, but try to maintain a consistent style within a file.